MoreLinq.Test/SliceTest.cs

```diff
@@ -125,5 +125,23 @@ public void TestSliceListOptimization()
             Assert.AreEqual(sliceCount, result.Count());
             Assert.IsTrue(result.SequenceEqual(Enumerable.Range(5, sliceCount)));
         }
+
+#if IREADONLY
```

Comment:
I don't think we define IREADONLY in the test project

Reply:
Sure we do. https://github.com/morelinq/MoreLINQ/blob/812045328654177044e67eb93212efa3a7de20a8/MoreLinq.Test/MoreLinq.Test.csproj#L22-L24

Reply:
Doh, my local clone was outdated. Sorry about that.